modules/openapi-generator/src/main/resources/elixir/model.mustache

```diff
@@ -16,7 +16,7 @@
   }
 end
 
-defimpl Poison.Decoder, for: {{&moduleName}}.Model.{{&classname}} do
+defimpl Jason.Decoder, for: {{&moduleName}}.Model.{{&classname}} do
```

Comment:
This results in the following error:

```
== Compilation error in file lib/openapi_petstore/model/_special_model_name_.ex ==
** (ArgumentError) Jason.Decoder is not a protocol
    (elixir 1.14.4) lib/protocol.ex:330: Protocol.assert_protocol!/2
    lib/openapi_petstore/model/_special_model_name_.ex:19: (file)
    (elixir 1.14.4) lib/kernel/parallel_compiler.ex:340: anonymous fn/5 in Kernel.ParallelCompiler.spawn_workers/7
 ```

Reply:
This might help in resolving this issue:

https://github.com/OpenAPITools/openapi-generator/pull/15746/files#diff-55ea12c96700d72d05d319301c9afd647715ce2ba84f43d949d78edbcd4fce5dL19-L36